tests/core/snapshots/test-required-fields.md_7.snap

```diff
@@ -0,0 +1,28 @@
+---
+source: tests/core/spec.rs
+expression: response
+---
+{
+  "status": 200,
+  "headers": {
+    "content-type": "application/json"
+  },
+  "body": {
+    "data": null,
```

Comment:
btw if I understand the spec right it should be `data: { innerMissing: null }` here. But I'm not sure is the problem in async_graphql or tailcall. 

Should we leave some todos related to this?

Reply:
I can check it with the graphql reference implementation and I will reply

Reply:
yeah data is `{ innerMissing: null }`

Reply:
Added a TODO, it can be solved to separate issue